webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -38,6 +38,27 @@ def login_required(
     return actual_decorator
 
 
+def superuser_required(
```

Comment:
Why move it to the ``crashstats`` app if it's only ever used in the ``manage`` app? I'd rather have it in ``manage/decorators.py`` than here.